api/swagger.yml

```diff
@@ -860,6 +860,7 @@ components:
         - blockstore_type
         - blockstore_namespace_example
         - blockstore_namespace_ValidityRegex
+        - pre_sign_support
```

Comment:
Does our config block `pre_sign`? I thought it's enabled by default.

Reply:
PreSign is enabled by default, meaning if you request for get physical address you will also get pre-sign url.
There is no block adapter configuration - but not all adapters supports this option, like: mem and local.
Added this code to identify it while testing. At runtime you will get an error.